blueocean-dashboard/src/main/js/components/Capability.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import update from 'react-addons-update';
```

Comment:
It would be interesting to know how much weight this adds to the bundles and if it's worth it Vs manually handling the update (shallow clones etc).